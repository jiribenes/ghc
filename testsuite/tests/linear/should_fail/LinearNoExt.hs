{-# LANGUAGE UnicodeSyntax #-}
module LinearNoExt where

type T = a ‚ä∏ a
