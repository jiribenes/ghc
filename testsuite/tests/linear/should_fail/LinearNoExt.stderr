
LinearNoExt.hs:4:12: error:
    Enable LinearTypes to allow linear functions
